testing/scripts/testing-script-template.md

```diff
@@ -16,17 +16,17 @@ Where should this be tested? The second paragraph describes on what level of the
 
 #### Proposed testing script
 
-| Step | Expected Behavior | 
-|-----|-----|
-| 1. What needs to happen to get from the beginning to the end? Steps are numbered. | What happens if things go correctly? Expected behavior describes successful checkpoints for this step in the script.|
-| 2.  | |
-| 3.  | | 
-| 4.  | |
-| 5.  | |
-| 6.  | |
-| 7.  | |
-| 8.  | |
-| 9.  | | 
-| 10. | | 
-
-Scripts can have as many or as few steps as needed to complete the task. Some steps may not need expected behavior, but try to add expected behavior as it can help implementation.
\ No newline at end of file
+| Step                                                                              | Expected Behavior                                                                                                    |
```

Comment:
Maybe we can add a note explaining the expected indentation of the table instead of having multiple spaces here. As it is right now, people using this template will have to manually remove spaces. I'm not sure if the pre-commit hook fixes this...

Reply:
The pre-commit _should_ fix this automatically (in fact I think these spaces were added by the pre-commit hook) so will leave it like this and if it causing issues we can add better rules for md tables or add comments